README.md

```diff
@@ -53,54 +52,57 @@ You interact with the API using a LuminosoClient object, which sends HTTP
 
 ```python
->>> from luminoso_api import LuminosoClient
->>> proj = LuminosoClient.connect('/projects/account_id/my_project_id',
-                                  username='my_username')
-Password for my_username: [here you enter your password]
->>> proj.get('terms')
-[lots of terms and vectors here]
+from luminoso_api import LuminosoClient
+project = LuminosoClient.connect('/projects/my_project_id', token='my_token')
+
+# And then, for instance:
+docs = project.get('docs', limit=10)
 ```
 
-Or you can connect using an existing API token:
+Instead of specifying the token when connecting, you can also use the
+LuminosoClient to save a token to a file, at which point you can connect
+without having to specify a token.
 
 ```python
 from luminoso_api import LuminosoClient
-proj = LuminosoClient.connect('/projects/account_id/my_project_id',
-                              token='my-api-token-here')
+LuminosoClient.save_token(token='my_token')
+project = LuminosoClient.connect('/projects/my_project_id')
+docs = project.get('docs', limit=10)
 ```
 
-You can even save your API token to a file on your computer and load it
-automatically, so that you don't have to specify any credentials:
+There is also a method, provided temporarily to ease the transition from the v4
+API, that allows you to connect with a username and password:
 
 ```python
 from luminoso_api import LuminosoClient
-client = LuminosoClient.connect(token='my-api-token-here')
-# This will save a non-expiring token, regardless of whether you are currently
-# using that token or some other token.
-client.save_token()
-# Now you can exit Python, restart your computer, etc., and your token will
-# still be saved when you come back.
-proj = LuminosoClient.connect('/projects/account_id/my_project_id')
+project = LuminosoClient.connect_with_username_and_password('/projects/my_project_id', username='my_username')
 ```
 
-When you connect without specifying a URL, the URL will be set to your default
-account_id under /projects:
+Note that the LuminosoClient will ensure that slashes are put in the right
+places, so that all of the following calls will go to the endpoint
+`https://daylight.luminoso.com/api/v5/projects/my_project_id/docs/`:
 
 ```python
->>> from luminoso_api import LuminosoClient
->>> projects = LuminosoClient.connect(username='testuser')
-Password: ...
->>> print(projects)
-<LuminosoClient for https://analytics.luminoso.com/api/v4/projects/lumi-test/>
+LuminosoClient.connect('/projects/my_project_id').get('docs')
+LuminosoClient.connect('/projects/my_project_id').get('/docs')
+LuminosoClient.connect('/projects/my_project_id').get('docs/')
+LuminosoClient.connect('/projects/my_project_id').get('/docs/')
+LuminosoClient.connect('/projects/my_project_id/').get('docs')
+LuminosoClient.connect('/projects/my_project_id/').get('/docs')
+LuminosoClient.connect('/projects/my_project_id/').get('docs/')
+LuminosoClient.connect('/projects/my_project_id/').get('/docs/')
```

Comment:
This is meant to be exhaustive: I'd wanted a genuine illustration, rather than a mere text description.    (Also note, for what it's worth, that this is already present in the V5_README.md file; the new README.md is a renamed, somewhat edited version of that, which GitHub is probably failing to figure out.)

Reply:
But do you actually need the slash before `projects`?  If so, then it's not entirely exhaustive.

Reply:
Well, OK, no...  We could add

> as will `LuminosoClient.connect('projects/my_project_id').get('docs')` and its variants.

after this block; or certainly if you have a rewrite of the section in general we can go with it instead.